packages/flutter/test/material/carousel_test.dart

```diff
@@ -1696,6 +1696,48 @@ void main() {
     expect(secondItemWidth, surfaceWidth * secondWeight / totalWeight);
   });
 
+  testWidgets('CarouselView.builder creates items lazily', (WidgetTester tester) async {
```

Comment:
Can we also add one more unit test for CarouselView.buildWeighted?

Reply:
Sure. will add..